compliance_checker/data/templates/ccheck.html.j2

```diff
@@ -8,6 +8,7 @@
   </div>
   <div class="col-md-12">
     <h3>{{testname}}</h3>
+    <a href={{cc_url}}>Reference Page</a>
```

Comment:
Can you make the test name in the line above be the hyperlink instead? And have it open in a new tab?

Reply:
Done, see the latest commit